From c7e89383dcdf9bfb6a51c0b22cacbf54bd44995d Mon Sep 17 00:00:00 2001
Message-Id: <c7e89383dcdf9bfb6a51c0b22cacbf54bd44995d@dist-git>
From: Peter Krempa <pkrempa@redhat.com>
Date: Fri, 26 Aug 2016 15:45:25 -0400
Subject: [PATCH] doc: clarify documentation for vcpu order

https://bugzilla.redhat.com/show_bug.cgi?id=1097930
https://bugzilla.redhat.com/show_bug.cgi?id=1224341

Make it clear that vcpu order is valid for online vcpus only and state
that it has to be specified for all vcpus or not provided at all.

Resolves: https://bugzilla.redhat.com/show_bug.cgi?id=1370043
(cherry picked from commit 54147fd9be31838904f6f2251510da91a1bd0ca7)
---
 docs/formatdomain.html.in | 7 ++++---
 1 file changed, 4 insertions(+), 3 deletions(-)

diff --git a/docs/formatdomain.html.in b/docs/formatdomain.html.in
index a838bff..c080bdf 100644
--- a/docs/formatdomain.html.in
+++ b/docs/formatdomain.html.in
@@ -564,11 +564,12 @@
         all disabled vcpus must be hotpluggable. Valid values are
         <code>yes</code> and <code>no</code>.
 
-        <code>order</code> allows to specify the order to add the vcpus. For
-        hypervisors/platforms that require to insert multiple vcpus at once
+        <code>order</code> allows to specify the order to add the online vcpus.
+        For hypervisors/platforms that require to insert multiple vcpus at once
         the order may be be duplicated accross all vcpus that need to be
         enabled at once. Specifying order is not necessary, vcpus are then
-        added in an arbitrary order.
+        added in an arbitrary order. If order info is used, it must be used for
+        all online vcpus.
 
         Note that hypervisors may create hotpluggable vcpus differently from
         boot vcpus thus special initialization may be necessary.
-- 
2.10.0

